src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Program.cs

```diff
@@ -157,70 +156,21 @@ private static void Execute(string[] args, bool isNoBuild, ILogger logger)
             return exitCode;
         }
 
-        private static IProjectContext GetProjectContext(string projectPath)
+        private static ProjectInformation GetProjectInformation(string projectPath)
         {
             var projectFileFinder = new ProjectFileFinder(projectPath);
             if (projectFileFinder.IsMsBuildProject)
             {
-                var frameworksInProject = TargetFrameworkFinder.GetAvailableTargetFrameworks(projectFileFinder.ProjectFilePath);
-
-                var nearestFramework = GetSuitableFrameworkFromProject(frameworksInProject);
-
-                return new MsBuildProjectContextBuilder()
-                    .AsDesignTimeBuild()
-                    .WithConfiguration("Debug")
-                    .WithProjectFile(projectFileFinder.ProjectFilePath)
-                    .WithTargetFramework(nearestFramework)
+                return new Internal.MsBuildProjectInformationBuilder(projectFileFinder.ProjectFilePath)
```

Comment:
Nit: add `using Microsoft.VisualStudio.Web.CodeGeneration.Tools.Internal` and remove this unnecessary qualifier.